frontend/src/utils/editorsList.js

```diff
@@ -1,9 +1,11 @@
+import { ID_EDITOR_URL, POTLATCH2_EDITOR_URL } from '../config';
+
 export function getEditors(filterList, customEditor) {
   let editors = [
     {
       label: 'iD Editor',
       value: 'ID',
-      url: 'http://www.openstreetmap.org/edit?editor=id&',
+      url: `${OSM_SERVER_URL}&`,
```

Comment:
OSM_SERVER_URL, ID_EDITOR_URL and POTLATCH2_EDITOR_URL are not being exported from `../config`